Server/tests/test_log_paths.py

```
@@ -0,0 +1,97 @@
+import os
+
+import pytest
+
+from utils.log_paths import resolve_log_dir
+
+
+class TestEnvOverride:
+    def test_override_wins_on_macos(self):
+        path = resolve_log_dir(
+            platform="darwin",
+            env={"UNITY_MCP_LOG_DIR": "/tmp/custom-logs"},
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/custom-logs"

(S108)